coordinape/coordinape
# Consolidate Event Triggers into EventManager

This is needed to cleanly manage the migrations of event triggers and
prevent them from duplicating logic with prod

test plan:
1. create a new user in the database with a new address and observe the
   creation of a profile entry.
2. In your console,set `export VERCEL_ENV=production`, rerun `vercel dev`
   and observe the `no handler configured for createProfile event` response
   message in the event trigger response.
3. Configure notification integrations to test out other endpoints, if
   you like
